ktor-client/ktor-client-tests/ktor-client-tests-dispatcher/iosArm32/src/TestIosArm32.kt

```diff
@@ -0,0 +1,22 @@
+@file:Suppress("INVISIBLE_MEMBER", "INVISIBLE_REFERENCE")
+
+package io.ktor.client.tests.utils.dispatcher
+
+import kotlinx.coroutines.*
+import kotlin.coroutines.*
+import platform.Foundation.*
+
+actual fun testSuspend(
+    context: CoroutineContext,
+    block: suspend CoroutineScope.() -> Unit
+): Unit = runBlocking {
+    val loop = coroutineContext[ContinuationInterceptor] as EventLoop
+
+    val task = launch { block() }
+    while (!task.isCompleted) {
+        val date = NSDate().addTimeInterval(1.0) as NSDate
```

Comment:
`addTimeInterval` is deprecated: https://developer.apple.com/documentation/foundation/nsdate/1591575-addtimeinterval

Reply:
There is no `addingTimeInterval` method in `kotlin-native` headers and it doesn't compile.